### ISSUE TITLE
Template versions out of date

### ISSUE THREAD

**Issue by cassidoo** [matt765/Tailcast#4]
I noticed that this theme uses Astro v1.4.4, and `framer-motion` is a few versions behind, as well as Tailwind.

**Comment by FriendlyUser** [comment]
I tried updating packages and saw no issues

  "dependencies": {
    "@astrojs/tailwind": "^3.1.1",
    "@fontsource/inter": "^4.5.14",
    "astro": "^2.1.7",
    "framer-motion": "^10.9.1",
    "tailwindcss": "^3.2.7"
  },

**Comment by matt765** [comment]
I updated all packages in repository

**Comment by cassidoo** [comment]
I noticed the "some checks are not successful" flag, any ideas what that's about?

**Comment by matt765** [comment]
It was an issue related to Github Pages build process that used Jekyll. I don't use Github Pages (I host website on Vercel from the beginning) so I disabled it along with Github Actions for this repository and the flag disappeared.

There is log file saved https://github.com/matt765/Tailcast/actions/runs/4639085395/jobs/8209553067
I looked through internet but I didn't find the solution for an error that is visible in this log file. If someone would want to use Github Pages and had this error I think it would be neccessary to contact AstroJS support.
